src/Microsoft.AspNetCore.SignalR.Common/Protocol/HubMethodInvocationMessage.cs

```diff
@@ -3,60 +3,37 @@
 
 using System;
 using System.Linq;
-using System.Runtime.ExceptionServices;
 
 namespace Microsoft.AspNetCore.SignalR.Protocol
 {
     public abstract class HubMethodInvocationMessage : HubInvocationMessage
     {
-        private readonly ExceptionDispatchInfo _argumentBindingException;
-        private readonly object[] _arguments;
-
         public string Target { get; }
 
-        public object[] Arguments
-        {
-            get
-            {
-                if (_argumentBindingException != null)
-                {
-                    _argumentBindingException.Throw();
-                }
-
-                return _arguments;
-            }
-        }
-
-        public Exception ArgumentBindingException => _argumentBindingException?.SourceException;
+        public object[] Arguments { get; }
 
-        protected HubMethodInvocationMessage(string invocationId, string target, ExceptionDispatchInfo argumentBindingException, object[] arguments)
+        protected HubMethodInvocationMessage(string invocationId, string target, object[] arguments)
             : base(invocationId)
         {
             if (string.IsNullOrEmpty(target))
             {
                 throw new ArgumentNullException(nameof(target));
             }
 
-            if ((arguments == null && argumentBindingException == null) || (arguments?.Length > 0 && argumentBindingException != null))
-            {
-                throw new ArgumentException($"'{nameof(argumentBindingException)}' and '{nameof(arguments)}' are mutually exclusive");
-            }
-
             Target = target;
-            _arguments = arguments;
-            _argumentBindingException = argumentBindingException;
+            Arguments = arguments;
```

Comment:
Check arguments is not null

Reply:
We allowed `null` before, and the protocols set it to null when there are no arguments. I can look at changing that though.